### PR TITLE
Entity script status + reload

### PR DESCRIPTION
**Pull request by huffman**

### CODE REVIEW

File: assignment-client/src/scripts/EntityScriptServer.cpp [hunk 1]
```diff
@@ -25,6 +25,8 @@
 
 #include "../entities/AssignmentParentFinder.h"
 
+const size_t UUID_LENGTH_BYTES = 16;
```
Comment (Atlante45) [inline assignment-client/src/scripts/EntityScriptServer.cpp]: Pretty sure we already have a constant for that somewhere, no?

Comment (birarda) [inline assignment-client/src/scripts/EntityScriptServer.cpp]: yes - `const int NUM_BYTES_RFC4122_UUID = 16;`

File: assignment-client/src/scripts/EntityScriptServer.cpp [hunk 2]
```diff
@@ -52,10 +54,39 @@ EntityScriptServer::EntityScriptServer(ReceivedMessage& message) : ThreadedAssig
...
+        checkAndCallPreload(entityID, true);
+    }
+}
+
+void EntityScriptServer::handleEntityScriptGetStatusPacket(QSharedPointer<ReceivedMessage> message, SharedNodePointer senderNode) {
+    MessageID messageID;
+    message->readPrimitive(&messageID);
+    auto entityID = QUuid::fromRfc4122(message->read(UUID_LENGTH_BYTES));
+
+    // TODO(Huffman) Get Status
```
Comment (Atlante45) [inline assignment-client/src/scripts/EntityScriptServer.cpp]: I bet this will still be there in 2021. =)

Comment (huffman) [inline assignment-client/src/scripts/EntityScriptServer.cpp]: Already fixed on my local branch :D

File: assignment-client/src/scripts/EntityScriptServer.cpp (hunk 1 above)
Comment (huffman) [inline assignment-client/src/scripts/EntityScriptServer.cpp]: Ok, I will use that, thanks!

File: libraries/networking/src/EntityScriptClient.cpp [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+void EntityScriptClient::handleNodeClientConnectionReset(SharedNodePointer node) {
+    // a client connection to a Node was reset
+    // if it was an EntityScriptServer we need to cause anything pending to fail so it is re-attempted
+
+    if (node->getType() != NodeType::EntityScriptServer) {
+        return;
+    }
+
+    //qCDebug(entity_script_client) << "EntityScriptClient detected client connection reset handshake with Asset Server - failing any pending requests";
```
Comment (Atlante45) [inline libraries/networking/src/EntityScriptClient.cpp]: Want to keep that?

**Pull request merged**